packages/devtools_app/lib/src/debugger/controls.dart

```diff
@@ -60,20 +61,26 @@ class _DebuggingControlsState extends State<DebuggingControls>
     @required bool isPaused,
     @required bool resuming,
   }) {
+    final isSystemIsolate = controller.isolateRef.isSystemIsolate;
```

Comment:
make this a getter in this class since it is repeated

Reply:
Done.